easybuild/easyconfigs/y/Yade/Yade-1.20.0-foss-2016a.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'CMakeMake'
+
+name = 'Yade'
+version = '1.20.0'
+
+homepage = 'https://yade-dem.org/doc/'
+description = """ Yade is an extensible open-source framework for discrete numerical models, 
+ focused on Discrete Element Method. The computation parts are written in c++ using flexible object model, 
+ allowing independent implementation of new alogrithms and interfaces. 
+ Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+suffix = '-Python-%(pyver)s'
```

Comment:
@jrbosch you'll need to remove this easyconfig, since you added one that includes the `-Python` versionsuffix...